operatorcourier/cli.py

```diff
@@ -19,6 +21,11 @@ def __init__(self):
         pass
 
     def parse(self):
+        def version():
+            with open('setup.py', 'r') as f:
```

Comment:
This won't work, `setup.py` is not a part of installed package, this works only from repository locally.

```bash
$ virtualenv venv-3
$ source venv-3/bin/activate
$ pip install git+https://github.com/jeremylinlin/operator-courier.git@add-version-arg-to-cli
$ operator-courier -v                                                                                                    
[Errno 2] No such file or directory: 'setup.py'
```

Reply:
Please consult this: https://packaging.python.org/guides/single-sourcing-package-version/